tests/ga/test_update.py

```diff
@@ -2919,16 +2919,53 @@ def test_it_should_mark_outdated_goal_states_on_service_restart_when_host_ga_plu
     def test_it_should_clear_the_timestamp_for_the_most_recent_fast_track_goal_state(self):
         data_file = self._prepare_fast_track_goal_state()
 
-        if HostPluginProtocol.get_fast_track_timestamp() is None:
+        if HostPluginProtocol.get_fast_track_timestamp() is timeutil.create_timestamp(datetime.min):
             raise Exception("The test setup did not save the Fast Track state")
 
         with patch("azurelinuxagent.common.conf.get_enable_fast_track", return_value=False):
             with mock_wire_protocol(data_file) as protocol:
                 with mock_update_handler(protocol) as update_handler:
                     update_handler.run()
 
-        self.assertIsNone(HostPluginProtocol.get_fast_track_timestamp(), "The Fast Track state was not cleared")
+        self.assertEqual(HostPluginProtocol.get_fast_track_timestamp(), timeutil.create_timestamp(datetime.min),
+            "The Fast Track state was not cleared")
 
+    def test_it_should_default_fast_track_timestamp_to_datetime_min(self):
+        data = DATA_FILE_VM_SETTINGS.copy()
+        data["vm_settings"] = "hostgaplugin/vm_settings-fabric-no_extension_manifests.json"
```

Comment:
are the extension manifests relevant to the test case?

Reply:
No, but I'm using an extensions config with no certificates to get around a bug in the test code (the permissions issue on the certificate file we talked about). The default `vm_settings.json` fails in `_check_certificates` when doing the fetch.

Reply:
yes, that is ok for goal_state_**no_certs**.xml, but why vm_settings-fabric-**no_extension_manifests**.json? you could have left the extension manifest in the test data and just remove the certificate properties.

or rename the file not to draw attention to the manifests, since they are irrelevant. vm_settings-fabric.json? (though test data without extension manifests looks funny)